sklearn/utils/sparsefuncs.py

```diff
@@ -86,14 +91,14 @@ def mean_variance_axis(X, axis):
 
     if isinstance(X, sp.csr_matrix):
         if axis == 0:
-            return _csr_mean_var_axis0(X)
+            return _csr_mean_var_axis0(X, ddof=ddof)
```

Comment:
We might have to check if ddof is not too large, otherwise results could be meaningless (< 0) or division by zero error.

Reply:
Checking that ddof<0 seems reasonable to me, but I don't really know what a good error message would be for a ddof that results in a division by zero error. Any suggestions?